README.md

```diff
@@ -40,6 +40,7 @@ For documentation, please visit [Dell Terraform Providers documentation](https:/
 ## Dell Terraform Providers
 * [Dell Terraform Provider for Powerflex](https://github.com/dell/terraform-provider-powerflex)
```

Comment:
Please change the platform names to PowerFlex, PowerMax, PowerStore

Reply:
@shenda1  updated